primo/utils/domain_validators.py

```diff
@@ -42,3 +45,25 @@ def _in_range(val):
         raise ValueError(f"Value {val} lies outside the admissible range [{lb}, {ub}]")
 
     return _in_range
+
+
+def validate_mobilization_cost(data: dict):
+    """
+    Validates the mobilization cost data
+
+    Parameters
+    ----------
+    data : dict
+        key => number of wells, value => cost of plugging
+
+    Returns
+    -------
+    data
+        Validated mobilization cost data.
+    """
+    for k in range(1, len(data) + 1):
```

Comment:
This assumes that the mobilization data has an entry for every increment (i.e., 1, 2, 3, 4, 5) and an input of (1 : 2500 ,4 : 2300 ,7 : 2000 ,9 : 1800) would fail (looking at cost per well in this case). Do we have a way to automatically populate for folks who just have certain costs at thresholds?

Reply:
(i.e. a function that takes {1 : 2500 ,4 : 2300 ,7 : 2000 , 9 : 1800} and turns it into {1: 2500, 2:2500, 3: 2500, 4: 2300, 5: 2300 ,...,})

Reply:
I think it is OK to leave as is for now: We can update once we start getting realistic mobilization costs from end users